src/Stepper/Step.tsx

```diff
@@ -38,25 +39,26 @@ const Step: FunctionComponent<Props> = ({
   theNextActive,
   vertical
 }) => {
-  const { color = '#d63426', icon, label, className } = step
+  const {color = '#d63426', icon, label, ariaLabel, className} = step
```

Comment:
Formatting preference would be to kee[ the spaces between the brackets and attributes here `const { color = '#d63426', icon, label, ariaLabel, className } = step`